Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:latest
```

Comment:
want to lock that down more with a version? I believe we use node 20 LTS. Would be good to have some consistency there

Reply:
Good call